dev/com.ibm.ws.jdbc/resources/com/ibm/ws/rsadapter/resources/IBMDataStoreAdapterNLS.nlsprops

```diff
@@ -366,6 +366,20 @@ PROP_SET_ERROR.useraction=Verify that the value that is specified is valid for t
 # 8060 deleted
 # 8065 deleted
 
+# 8066E
+# {0}         The invalid attribute value
+# {1}         The allowed attribute values
+# Do not translate: identifyException, as
+8066E_IDENTIFY_EXCEPTION_INVALID_TARGET=DSRA8066E: Invalid value ''{0}'' specified for the identifyException ''as'' attribute. Allowed values are: {1}
+8066E_IDENTIFY_EXCEPTION_INVALID_TARGET.explanation=An unknown value was specified for the identifyException configuration element.
+8066E_IDENTIFY_EXCEPTION_INVALID_TARGET.useraction=Update the configuration to specify one of the allowed values.
+
+# 8067E
+# Do not translate: errorCode, sqlState, identifyException
+8067E_IDENTIFY_EXCEPTION_ERRCODE_SQLSTATE=DSRA8067E: Must specify exactly one of the ''errorCode'' or ''sqlState'' attributes on the identifyException element.
```

Comment:
Must specify exactly one of the ''errorCode'' or ''sqlState'' attributes on the identifyException element.
--->
Exactly one of either the ''errorCode'' or ''sqlState'' attributes must be specified on the identifyException element.